src/common/Icons.js

```diff
@@ -20,6 +20,7 @@ export const IconPrivateChat = (props: Object) => <Feather name="mail" {...props
 export const IconDownArrow = (props: Object) => <Feather name="chevron-down" {...props} />;
 export const IconGoogle = (props: Object) => <IoniconsIcon name="logo-google" {...props} />;
 export const IconGitHub = (props: Object) => <Feather name="github" {...props} />;
+export const IconAzureAD = (props: Object) => <IoniconsIcon name="logo-windows" {...props} />;
```

Comment:
Maybe call it `IconWindows` even though we use it for Azure.
The logo for Azure is different, but I think Windows' logo is a reasonable replacement.